cla-backend-go/v2/sign/service.go

```diff
@@ -98,7 +98,9 @@ func (in *requestCorporateSignatureOutput) toModel() *models.CorporateSignatureO
 
 func validateCorporateSignatureInput(input *models.CorporateSignatureInput) error {
 	if input.SendAsEmail {
-		if input.AuthorityName == "" {
+		log.Debugf("input.AuthorityName validation %s", input.AuthorityName)
+		if strings.TrimSpace(input.AuthorityName) == "" {
```

Comment:
@dealako had recommended pattern checking in the swagger doc for field validations.

Reply:
Update the swagger field validation with proper pattern.